packages/component-library/draft/Kaizen/TitleBlock/NavigationButton.scss

```diff
@@ -27,6 +27,7 @@
 
   @include ca-media-tablet-and-up {
     color: $kz-color-wisteria-800;
+    padding-top: 17px;
```

Comment:
This was to nudge the vertically-centred menu buttons into alignment with the new heading, so that they sit on the same baseline (previously they had both been vertically centred, but the new font being bigger caused that to look weird).